src/pyird/utils/irdstream.py

```diff
@@ -370,44 +382,49 @@ def mkwspec(spec_m2,reference,save_path):
                 reference = hdu.data
 
                 id = self.fitsid[j]
-                save_path = self.anadir/('%s%d_m2.dat'%(prefix,id)) ##for mmf2
+                save_path = self.anadir/('%s%d_%s.dat'%(prefix,id,self.trace.mmf)) ##e.g. w41511_m2.dat
                 wspec = mkwspec(spec_m2,reference,save_path)
                 if self.info:
-                    print('dispcor: output spectrum= %s%d_m2.dat'%(prefix,id))
+                    print('dispcor: output spectrum= %s%d_%s.dat'%(prefix,id,self.trace.mmf))
                 #plot
                 show_wavcal_spectrum(wspec,alpha=0.5)
         else:
-            hdu = pyf.open(self.anadir/('%s_mmf12.fits'%(self.streamid)))[0]
+            hdu = pyf.open(self.anadir/('%s_mmf12_%s.fits'%(self.streamid,self.band)))[0]
```

Comment:
self.trace.mmf?

Reply:
Thank you for pointing this out!! I just missed changing this part, so it was not intentional.
I will fix the naming convention to {streamID}_{band}_{mmf}.fits for the master file.